experimental/dds/tree2/src/feature-libraries/editable-tree/editableTree.ts

```diff
@@ -62,6 +63,13 @@ import { ProxyTarget } from "./ProxyTarget";
 
 const editableTreeSlot = anchorSlot<EditableTree>();
 
+export interface DetachedFieldCache {
```

Comment:
I think these types could be declared in utilities.ts instead to avoid exporting them from any file.

Reply:
Moved `DetachedFieldCache` and `detachedFieldSlot` to `utilities.ts`.